zerver/migrations/0465_rename_followup_day_email_templates.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.2 on 2023-07-11 10:45
+
+import orjson
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+WELCOME = 1
+
+
+def update_for_followup_day_email_templates_rename(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    ScheduledEmail = apps.get_model("zerver", "ScheduledEmail")
+    scheduled_onboarding_emails = ScheduledEmail.objects.filter(type=WELCOME)
+
+    for email in scheduled_onboarding_emails:
+        data = orjson.loads(email.data)
+        template = data["template_prefix"]
+        if template == "followup_day1":
+            data["template_prefix"] = "account_registered"
+            email.data = orjson.dumps(data).decode()
+        elif template == "followup_day2":
+            data["template_prefix"] = "onboarding_zulip_topics"
+            email.data = orjson.dumps(data).decode()
+
+    ScheduledEmail.objects.bulk_update(scheduled_onboarding_emails, fields=["data"])
+
+
+def revert_followup_day_email_templates_rename(
```

Comment:
It was simple enough to write the reverse loop, so I went ahead and did that here.